salt/modules/jenkins.py

```diff
@@ -426,3 +426,31 @@ def get_job_config(name=None):
 
     job_info = server.get_job_config(name)
     return job_info
+
+def plugin_installed(name=None):
+    '''
+    Return if the plugin is installed for the provided plugin name.
+
+    :param name: The name of the parameter to confirm installation.
+    :return: True if plugin exists, False if plugin does not exist.
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' jenkins.plugin_installed pluginName
+
+    '''
+
+    if not name:
```

Comment:
What if you just made `name` an arg instead of a kwarg? That was you wouldn't have to do this check. If the arg is missing, Salt will handle the error. (So just pass `name` in the function definition instead of `name=None`.)